vm.js

```diff
@@ -3248,6 +3249,394 @@ Object.subclass('Squeak.Primitives',
+            /* */
+            //MIDI Primitives (520-539)
+            521: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIClosePort', argCount); },
+            522: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIGetClock', argCount); },
+            523: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIGetPortCount', argCount); },
+            524: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIGetPortDirectionality', argCount); },
+            525: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIGetPortName', argCount); },
+            526: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIOpenPort', argCount); },
+            527: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIParameterGetOrSet', argCount); },
+            528: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIRead', argCount); },
+            529: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('MIDIPlugin', 'primitiveMIDIWrite', argCount); },
+            // 530-539: reserved for extended MIDI primitives
+            // Sound Codec Primitives",
+            550: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('ADPCMCodecPlugin', 'primitiveDecodeMono', argCount); },
+            551: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('ADPCMCodecPlugin', 'primitiveDecodeStereo', argCount); },
+            552: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('ADPCMCodecPlugin', 'primitiveEncodeMono', argCount); },
+            553: function (index, argCount, primMethod, primHandler) { return primHandler.namedPrimitive('ADPCMCodecPlugin', 'primitiveEncodeStereo', argCount); },
+            // External primitive support primitives (570-574)
+            // 570: function (index, argCount, primMethod, primHandler) { primHandler.primitiveFlushExternalPrimitives(argCount); },
+            571: function (index, argCount, primMethod, primHandler) { return primHandler.primitiveUnloadModule(argCount); },
+            572: function (index, argCount, primMethod, primHandler) { return primHandler.primitiveListBuiltinModule(argCount); },
+            573: function (index, argCount, primMethod, primHandler) { return primHandler.primitiveListLoadedModule(argCount) }
+        };
+
+        if(this.oldPrims) {
+            // File Primitives (150-169)
```

Comment:
ehm. actually: https://github.com/bertfreudenberg/SqueakJS/blob/master/vm.js#L3581

should we fix it downstream?